lib/cib/cib_utils.c

```diff
@@ -771,9 +771,6 @@ cib__signon_query(pcmk__output_t *out, cib_t **cib, xmlNode **cib_object)
         cib__clean_up_connection(&cib_conn);
     }
 
-    if (*cib_object == NULL) {
-        return pcmk_rc_no_input;
-    }
```

Comment:
I think the real intention here was:

    if ((rc == pcmk_rc_ok) && (*cib_object == NULL)) {

I doubt that's possible in practice but it seems like a reasonable failsafe

Reply:
Yeah I think that's better